armory/docker/management.py

```diff
@@ -30,6 +30,8 @@ def __init__(
         host_paths = paths.HostPaths()
         docker_paths = paths.DockerPaths()
 
+        envs["ARMORY_VERSION"] = armory.__version__
+
```

Comment:
Having this as an override to what the evaluator adds seems both redundant and hard to follow. I would recommend you set this in one or the other.